pkg/operator/apis/monitoring/v1/common_types.go

```diff
@@ -104,6 +104,8 @@ func relabelingsForSelector(selector metav1.LabelSelector, crd interface{}) ([]*
 				SourceLabels: prommodel.LabelNames{objectLabelPresent + sanitizeLabelName(exp.Key)},
 				Regex:        relabel.MustNewRegexp("true"),
 			})
+		default:
```

Comment:
TODO for myself (or anyone else): Double check if this is for sure all cases in op.Operator.